cluster/manifests/deployment-service/status-service-rbac.yaml

```diff
@@ -22,44 +22,6 @@ rules:
     resources:
       - cdpdeploymenttasks
     verbs: ["get", "list", "watch"]
-  - apiGroups:
```

Comment:
Why all these deletions were done? Just curious because I don't know.

Reply:
These permissions become redundant when we give status-service the `readonly` role.

Reply:
Ah, got it 🙂